prow/plugins/verify-owners/verify-owners.go

```diff
@@ -157,12 +182,11 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, pre *github.Pul
 		}
 		log.Debugf("Creating a review for %d %s file%s.", len(wrongOwnersFiles), ownersFileName, s)
 		var comments []github.DraftReviewComment
-		for errFile, errMsg := range wrongOwnersFiles {
-			resp := errMsg
+		for errFile, err := range wrongOwnersFiles {
 			comments = append(comments, github.DraftReviewComment{
 				Path:     errFile,
-				Body:     resp,
-				Position: 1,
+				Body:     err.message,
+				Position: err.line,
```

Comment:
Line number here is relative to the diff, not absolute line number in the file AFAIR

Reply:
Yeah, you're right! Hmm then it makes it really complicated...
Is it possible to make a review with no particular line number? Better nothing than something wrong :-(

Reply:
What you had before (line number 1) should be OK -- but you could look at how the comments are made in the `/lint` plugin, since that one posts on the correct line

Reply:
How about now?